db/db.py

```diff
@@ -0,0 +1,28 @@
+import pymongo
+
+class db:
+
+    def __init__(self, ip="127.0.0.1", port=27017, dbname="pastepwn", collectionname="col"):
```

Comment:
The collection name should represent the data it contains. Something like "pastes" should be great.